packages/components/src/form-token-field/styles.ts

```diff
@@ -27,6 +27,7 @@ const deprecatedPaddings = ( {
 
 export const TokensAndInputWrapperFlex = styled( Flex )`
 	padding: 7px;
+	box-sizing: border-box;
```

Comment:
Just in case, should we replace it with this mixin to propagate the `box-sizing` value to the child elements?

Reply:
Sure! I think it should be safe because the [sidebar had box-sizing resets](https://github.com/WordPress/gutenberg/blob/33bf984897683adfc4582a29adb079cf6ee0ab59/packages/edit-post/src/style.scss#L59-L65) this entire time, like you noted 👍